### PR TITLE
Enable passing of more Pushover API parameters.

### PR DESCRIPTION
**Pull request by jlrgraham**
Allow users to specify more of the parameters allowed by the [Pushover API](https://pushover.net/api).  In particular, add support for the embedded URL and HTML formatting options.

### CODE REVIEW

File: HANDBOOK.md [hunk 1]
```diff
@@ -2281,6 +2281,21 @@ NOTE: `callback` is an optional URL for pushover to [ack messages](https://pusho
 | `title`       |   O    | application title (dflt: pushover dflt) |
 | `priority`    |   O    | priority. (dflt: pushover setting)     |
 
+Users can enable Pushover's [HTML sytling](https://pushover.net/api#html) or [Supplementary URLs](https://pushover.net/api#urls) support in messages by adding the `html`, `url`, and `url_title` keys to the data object:
```
Comment (amotl) [inline HANDBOOK.md:2284]:
```suggestion
Users can enable Pushover's [HTML styling](https://pushover.net/api#html) or [Supplementary URLs](https://pushover.net/api#urls) support in messages by adding the `html`, `url`, and `url_title` keys to the data object:
```

**Review by amotl — CHANGES_REQUESTED** [review]
Dear @jlrgraham,

thanks a stack for your contribution. I think it will be good to go and I am looking forward to cut another release after merging.

On the feedback side, I spotted a minor typo where I am humbly asking you for correction.

With kind regards,
Andreas.

File: HANDBOOK.md (hunk 1 above)
Comment (jlrgraham) [reply HANDBOOK.md]: My apologies, thanks for the catch, updated.

**Review by amotl — APPROVED** [review]

**Comment by amotl** [comment]
Dear Justin,

before merging, I've just added a corresponding test case to cover the added code. Thank you again.

With kind regards,
Andreas.

**Comment by jlrgraham** [comment]
@amotl, fantastic, many thanks.  I'll try to make sure that I get appropriate tests and coverage for future submissions.  Best, -Justin

**Pull request merged**